jsonschema-core.xml

```diff
@@ -291,22 +326,27 @@
             <t>
                 Additionally, the "application/schema+json" media type supports two
                 fragment identifier structures: plain names and JSON Pointers.
+                The "application/schema-instance+json" media type supports one
+                fragment identifier structure: JSON Pointers.
+            </t>
+            <t>
                 The use of JSON Pointers as URI fragment identifiers is described in
                 <xref target="RFC6901">RFC 6901</xref>.
-                Fragment identifiers matching the JSON Pointer syntax, including
-                the empty string, MUST be interpreted as JSON Pointer fragment
+                For "application/schema+json", fragment identifiers matching the JSON Pointer
+                syntax, including the empty string, MUST be interpreted as JSON Pointer fragment
                 identifiers.
```

Comment:
I understood this paragraph as applicable to both "application/schema+json" and "application/schema-instance+json", am I wrong? Otherwise, why adding the `For ""application/schema+json",` restriction?

Reply:
`application/schema-instance+json` only has one fragment identifier syntax, so this sentence is not necessary for it.  It's also not incorrect, so you're not wrong.  I'll try to tweak the wording.

I suspect that after all of the `schema-instance+json` stuff goes in, we'll want to rework the overall flow of the document _before_ publishing draft-07, so I don't want to get too hung up on the wording.  But I will update this with some improvements.

Reply:
> `application/schema-instance+json` only has one fragment identifier syntax, so this sentence is not necessary for it. It's also not incorrect, so you're not wrong. I'll try to tweak the wording.

I see, it makes sense now. Your clarification commit helps.